torchtext/__init__.py

```diff
@@ -4,7 +4,7 @@
 from . import vocab
 from . import legacy
 
-__version__ = '0.4.0'
```

Comment:
What is the purpose of this PR? The title hints that it's about IMDB, right?

Reply:
Move the version back to '0.4.0'